component/loki/source/kafka/kafka_test.go

```diff
@@ -59,6 +59,8 @@ func TestSASLRiverConfig(t *testing.T) {
 	var args Arguments
 	err := river.Unmarshal([]byte(exampleRiverConfig), &args)
 	require.NoError(t, err)
+
+	require.NotEqual(t, "password", args.Authentication.SASLConfig.Password.String())
```

Comment:
I personally think a `require.Equal` would give the reader of the test a better understanding of what the expected output is.

Reply:
Yes, you're right. I realized it after pushing the changes. I'll do it, thanks for pointing it out.